contents/blog/real-user-monitoring.md

```diff
@@ -0,0 +1,118 @@
+
+## Why do real user monitoring?
+
+Real user monitoring is important because it helps you understand your current user experience, discover ways to improve it, and manage the risk of future issues.
+
+![Why](../images/blog/real-user-monitoring/why.png)
+
+### 1. Understand current experience
+
+The first reason to do real user monitoring is to ensure your product performs well. Providing a quality experience matters. To make this happen, you must have visibility into what the current experience is like for users.
+
+This means understanding query performance, core web vitals, front and backend error rates, usage challenges, and more. These impact a user’s ability to accomplish their goals with your product. Issues doing this create a negative experience that hurts your business overall.
+
+Understanding and maintaining a high-quality user experience is critical to creating a successful product. Real user monitoring creates visibility into the user experience.
+
+### 2. Improve experience
+
+There are always endless potential improvements to your product. Each of them (hopefully) creates some amount of positive impact on users’ experience. To get the largest impact on what you build, real user monitoring captures metrics to help prioritize these improvements. For example:
+
+- A slow query might not matter for a free user but might be a top priority for a large potential customer in a trial stage.
+- Multiple users complain about slow queries, but you don’t know which ones or how slow they are without some tracking on them.
+- A user complains a feature "sucks," but you don’t know what this means. It could mean slow page speed, poor design, a bug, or a lack of knowledge.
+
+Real user monitoring helps you get the information to understand and prioritize the issues impacting user experience, whether it is query speeds or session replays. Often, this data is key to solving the issue.
+
+### 3. Manage risk
+
+The third and final reason to do real user monitoring is it helps manage risk. It does this by identifying issues and enabling plans to limit them. It enables companies to make promises about their experience quality, and back those promises with data.
+
+Using software comes with risks. For example, a payment processor being down could lead to millions in lost revenue for companies relying on it. Having monitoring in place helps limit these risks proactively. When working with larger companies, risk management is critical.
+
+Companies want to know the products they use have high uptime, and won’t be slow or buggy. This is often solidified in a service-level agreement, or SLA, which is a contract defining the expected service levels an application must provide (such as uptime and performance). Real user monitoring measures metrics important to SLAs, helping companies conform to them.
+
+Real user monitoring creates trust, which is both useful for internal predictions and external promises.
+
```

Comment:
same comment as above. I dont think we need this. I think our target audience is already aware of the benefits from the earlier explanation in this blog. I dont think we need to go into more detail

Reply:
We don't need what? The whole "why" section?

Reply:
Annoyingly github doesn't show which section of code of I highlighted when making this comment.

But yes, I don't think we need the entire "Why do real user monitoring?" section. We cover it enough in the intro and in the section " Real user monitoring in detail"

Reply:
Hmm, if this is a tutorial it makes sense. If it is blog, I need this section as it is the "meat" of the post.

I'm trying to do something [like this](https://posthog.com/blog/feature-flag-benefits-use-cases) for monitoring basically.